### PR TITLE
Sync PetParsing to WoD

### PR DESCRIPTION
**Pull request by funjoker**
Sync PetParsing to WoD

Thx to @mdx7

### CODE REVIEW

File: WowPacketParser/Enums/PetStableResult.cs [hunk 1]
```diff
@@ -10,4 +10,15 @@ public enum PetStableResult
         STABLE_ERR_EXOTIC       = 0x0B,                         // "you are unable to control exotic creatures"
         STABLE_ERR_STABLE       = 0x0C,                         // "Internal pet error"
     }
+
+    public enum PetStableResult732
```
Comment (Shauren) [inline WowPacketParser/Enums/PetStableResult.cs]: should move this one to 7_0 module project

File: WowPacketParserModule.V6_0_2_19033/Parsers/PetHandler.cs [hunk 2]
```diff
@@ -247,5 +247,34 @@ public static void HandlePetActionFeedback(Packet packet)
...
+        [Parser(Opcode.SMSG_PET_SLOT_UPDATED)]
+        public static void HandlePetSlotUpdated(Packet packet)
+        {
+            packet.ReadInt32("PetNumberA");
+            packet.ReadInt32("PetSlotA");
+            packet.ReadInt32("PetNumberB");
+            packet.ReadInt32("PetSlotB");
+        }
+
+        [Parser(Opcode.SMSG_PET_STABLE_RESULT, ClientVersionBuild.V6_0_2_19033, ClientVersionBuild.V7_3_2_25383)]
```
Comment (Shauren) [inline WowPacketParserModule.V6_0_2_19033/Parsers/PetHandler.cs]: are you sure about 7.3.2 being the version that changed enum?

Comment (mdX7) [reply WowPacketParserModule.V6_0_2_19033/Parsers/PetHandler.cs]: Yes, gonna check later again, when I'm home. But it's definitely not in < 7.x

Comment (mdX7) [reply WowPacketParserModule.V6_0_2_19033/Parsers/PetHandler.cs]: Nevermind. Enum is still the same. Missed a -1 at variable init and didn't notice other changes. Kinda stressed atm - sorry.

**Comment by funjoker** [comment]
WoD used new stabel system already.
So this should go to WoD

**Comment by mdX7** [comment]
yup. Seems fine now

**Pull request merged**